app/course/directives/assignTagTooltip/assignTagTooltip.css

```diff
@@ -44,7 +44,7 @@
 	border: 1px solid rgba(27,31,35,0.15);
 	border-radius: 3px;
```

Comment:
Your PR shouldn't involve touching this file, I suspect something went wrong git-ing, I'll look at it with you next time we're in the office.